src/index.html

```diff
@@ -7,12 +7,63 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <form
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__imac"
+      />
+      <h2 class="card__model">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="card__code">Product code: 195434</div>
+      <div class="rating-container">
+        <div class="stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```

Reply:
fixed